src/components/SetGuestUsername.vue

```diff
@@ -129,6 +132,12 @@ function toggleEdit() {
 		})
 	}
 }
+
+// One-way binding to parent component
+watch(guestUserName, (newValue) => {
+	emit('update:guest-username', newValue)
```

Comment:
It's prefilled from displayName watcher (reproducible with clear cache)

![Image](https://github.com/user-attachments/assets/cc2e4a47-194c-432d-8f86-a6c6f132404a)